readme.md

```diff
@@ -170,30 +174,29 @@ async def run():
     except ErrTimeout:
         print("Flush timeout")
 
-    await asyncio.sleep(1, loop=loop)
+    await asyncio.sleep(1)
 
     # Drain gracefully closes the connection, allowing all subscribers to
     # handle any pending messages inflight that the server may have sent.
     await nc.drain()
 
 if __name__ == '__main__':
     loop = asyncio.get_event_loop()
-    loop.run_until_complete(run())
+    loop.run_until_complete(run(loop))
```

Comment:
Should we remove `loop`? I don't think it's needed and it might be good not to encourage passing explicit loops around